provider/cluster/kube/client.go

```diff
@@ -189,11 +185,11 @@ func (c *client) Deploy(ctx context.Context, lid mtypes.LeaseID, group *manifest
 		}
 
 		for expIdx := range service.Expose {
-			expose := &service.Expose[expIdx]
-			if !shouldExpose(expose) {
+			expose := service.Expose[expIdx]
+			if !util.ShouldExpose(expose) {
 				continue
 			}
-			if err := applyIngress(ctx, c.kc, newIngressBuilder(c.log, c.settings, c.host, lid, group, service, expose)); err != nil {
+			if err := applyIngress(ctx, c.kc, newIngressBuilder(c.log, c.settings, c.host, lid, group, service, &service.Expose[expIdx])); err != nil {
```

Comment:
`&expose`?  hmm is there a reason why this takes a pointer?  generally prefer pass-by-value to contain any possible mutation.

Reply:
I'm not actually sure why we were passing around `*ServiceExpose`. I eliminated it in a few places in the codebase because it was causing some warnings.